vmdb/spec/models/miq_provision_virt_workflow_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "spec_helper"
+
+describe MiqProvisionVirtWorkflow do
+  context "#continue_request" do
+    let(:sdn) { "SysprepDomainName" }
+    let(:wf)  { FactoryGirl.create(:miq_provision_virt_workflow) }
+
+    before do
+      FactoryGirl.create(:user_admin)
+      MiqDialog.seed
```

Comment:
I got rid of this in other places, so there should be a way to do it without seeding